web/src/main/scala/net/wiringbits/components/pages/DashboardPage.scala

```diff
@@ -12,15 +13,16 @@ import slinky.core.facade.Fragment
   case class Props(ctx: AppContext, user: User)
 
   val component: FunctionalComponent[Props] = FunctionalComponent[Props] { props =>
+    val texts = I18nHooks.useMessages(props.ctx.$lang)
     Fragment(
       Container(
         margin = Container.EdgeInsets.bottom(16),
         child = Fragment(
-          Title("Dashboard Page"),
-          Subtitle(s"${AppStrings.welcome} ${props.user.name}")
+          Title(texts.dashboardPage),
+          Subtitle(s"${texts.welcome} ${props.user.name}")
```

Comment:
A nice details from this approach is that you can do `texts.welcome(props.user.name)` instead